packages/picker/test/index.ts

```diff
@@ -846,15 +848,15 @@ export function runPickerTests(): void {
             expect(el.value).to.equal('');
             expect(secondItem.selected).to.be.false;
 
-            secondItem.click();
+            /*secondItem.click();
```

Comment:
TODO this part times out

Reply:
Updated this from el.click() to coordinate-based mouse event simulation which now properly triggers the event propagation chain and more accurately mimics real user interaction